src/graphql/main/schema.graphql

```diff
@@ -974,6 +1000,11 @@ type Query {
     @deprecated(
       reason: "TODO: remove. we don't need a non authenticated version of this query. the users can only do the query while authenticated"
     )
+
+  """
+  Returns 1 Sat and 1 Usd Cent price for the given currency
+  """
+  realtimePrice(currency: DisplayCurrency = "USD"): RealtimePrice
```

Comment:
the return type should be mandatory (instead of optional)

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/2298/commits/220c4ae26f4ec62588809307892cf8559ce486ef